test/cli/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ target_use_datasources (find_deletions_cli_test FILES detect_breakends_shorted.v
 
 add_cli_test(detect_breakends_cli_test.cpp)
 add_dependencies (detect_breakends_cli_test "detect_breakends")
-target_use_datasources (detect_breakends_cli_test FILES converted_bam_shorted.sam)
+target_use_datasources (detect_breakends_cli_test FILES simulated.minimap2.hg19.coordsorted_cutoff.sam)
```

Comment:
This is code from https://github.com/seqan/iGenVar/pull/34 right ? Not sure if its a problem if it appears here as well. :)

Reply:
also in test/cli/detect_breakends_cli_test.cpp

Reply:
It's good that you note that again. I have to merge them beforehand. I changed the order of the output in some places and didn't want to have to touch everything twice ...